compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Env.hs

```diff
@@ -114,3 +115,6 @@ throwWithContext err = do
 
 withContext :: MonadGamma m => Context -> m b -> m b
 withContext ctx = local (set locCtx ctx)
+
+catchAndRethrow :: MonadGamma m => (Error -> Error) -> m b -> m b
+catchAndRethrow handler mb = catchError mb $ throwWithContext . handler
```

Comment:
hm I couldn't find it documented anywhere, but I think we want an empty newline at the end of all source files

```suggestion
catchAndRethrow handler mb = catchError mb $ throwWithContext . handler

```